Scripts/CI/README_Metadata_StyleCheck/entry.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import os
+import json
+import argparse
+import subprocess as sp
+
+# A set of category folder names in current sample viewer.
+# Only run the checks when a file path is within one of these category folders.
+categories = {
+    'analysis',
+    'display_information',
+    'editing',
+    'feature_layers',
+    'geometry',
+    'group_layers',
+    'hydrography',
+    'image_layers',
+    'kml',
+    'local_server',
+    'map',
+    'map_view',
+    'network_analysis',
+    'ogc',
+    'portal',
+    'raster',
+    'scene',
+    'search',
+    'symbology',
+    'tiled_layers',
+    'utility_network'
+}
+
+def run_mdl(readme_path: str):
+    print("**** markdownlint style checker (mdl) ****")
+    code = sp.call(f'mdl --style /style.rb "{readme_path}"', shell=True)
+    return code
+
+def run_style_check(dirname: str):
```

Comment:
I think it would be helpful to add some comments here as to why we add together `code1 + code2` and what the return value is used for?

Reply:
Added in 👍